docs/PfSense.md

```diff
@@ -0,0 +1,35 @@
+In order to boot PfSense via PXE,
+you need to do the following:
+
+1. Download ISO from https://nyifiles.pfsense.org/mirror/downloads/pfSense-CE-2.3.4-RELEASE-amd64.iso.gz
+   
+   You should also check the SHA256 hash:
+   `sha256 -c 610b2a8c696e1d1854845d553ce8540debaab9d78ed6c15ce83872eaeac9d05f pfSense-CE-2.3.4-RELEASE-amd64.iso.gz`
+   and compare it to the one in https://nyifiles.pfsense.org/mirror/downloads/pfSense-CE-2.3.4-RELEASE-amd64.iso.gz.sha256
+
+2. Extract the ISO contents to a directory called
+    `images/pfsense` in TFTP root:
+    `gzip -d pfSense-CE-2.3.4-RELEASE-amd64.iso.gz`
+    
+   In order to extract the contents, you need to mount
+   the ISO with following commands (example from FreeBSD):
+   `mount_cd9660 /dev/$(mdconfig -a -t vnode -f pfSense-CE-2.3.4-RELEASE-amd64.iso) /mnt`
```

Comment:
@pkubaj on Debian like systems it would be easier to use this: 
`mount -o loop pfSense-CE-2.3.4-RELEASE-amd64.iso /mnt` and then rsync.
It also mounts an ISO image to target directory  and creates a device node with name `loopX`.
I do not have mount_cd9660 command. 
I got some errors when running rsync: 

```
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1183) [sender=3.1.1]
```

Reply:
Considering that most of our target audience (and our validation team) use Linux, please provide examples using some modern, popular distro (e.g. Debian). FreeBSD examples are also welcome, but shouldn't be the only one - sorry :)

Reply:
@miczyg1 it looks that rsync couldn't transfer some files attributes exactly (like read/write/execute permissions).

Reply:
@mek-x is right and this shouldn't really matter.